### PR TITLE
Small copy changes to TestUtils and Perf summaries

### PR DESCRIPTION
**Pull request by chrisbolin**
TestUtils - removed redundant "unminified" callout (it's on the line above). Perf - made it a noun clause (like most other items) and cleaning up grammar.

### CODE REVIEW

File: docs/docs/10-addons.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The React add-ons are a collection of useful utility modules for building React
 
 The add-ons below are in the development (unminified) version of React only:
 
-- [`TestUtils`](test-utils.html), simple helpers for writing test cases (unminified build only).
-- [`Perf`](perf.html), for measuring performance and giving you hint where to optimize.
+- [`TestUtils`](test-utils.html), simple helpers for writing test cases.
+- [`Perf`](perf.html), a performance profiling tool for finding optimizations opportunities.
```
Comment (bhamodi) [inline docs/docs/10-addons.md]: Nit pick: `optimizations` should probably just be `optimization` here.

Comment (chrisbolin) [inline docs/docs/10-addons.md]: You're totally right! I'll fix that

**Comment by zpao** [comment]
👍 Thanks! Just fix that one pluralization issue and make sure you squash to a single commit, then we can merge.

**Comment by chrisbolin** [comment]
@zpao the pluralization issue is fixed

**Comment by zpao** [comment]
Thanks!

**Pull request merged**